client/verta/verta/operations/monitoring/profilers.py

```diff
@@ -227,20 +343,23 @@ def create(self, name, model_version):
         reference._set_client(self._client)
         return reference
 
+    # TODO: Document behavior which happens if this is a deployed profiler?
+    # TODO: Document the error raised by a failure
+    def delete(self, profiler_reference):
+        """Delete the provided profiler reference.
 
-    @staticmethod
-    def enable(profiler_reference, monitored_entity):
-        return profiler_reference.enable(monitored_entity)
-
-    @staticmethod
-    def disable(profiler_reference, monitored_entity):
-        return profiler_reference.disable(monitored_entity)
+        Instructs Verta Services to delete .
```

Comment:
Extended summaries may not be necessary for some of these! Unless `pydocstyle` favors them.